pyproject.toml

```diff
@@ -110,3 +95,20 @@ show_error_codes = true
 module = ['tests.*']
 disallow_untyped_defs = false
 strict_equality = false
+
+[tool.ruff]
+select = ["A", "B", "C", "E", "F", "G", "I", "N", "Q", "S", "T", "W", "ANN", "ARG", "BLE", "COM", "DJ", "DTZ", "EM", "ERA", "EXE", "FBT", "ICN", "INP", "ISC", "NPY", "PD", "PGH", "PIE", "PL", "PT", "PTH", "PYI", "RET", "RSE", "RUF", "SIM", "SLF", "TCH", "TID", "TRY", "UP", "YTT"]
```

Comment:
Could also just use `select = ["ALL"]`. However, note that it'll implicitly enable all the new rules of the future versions of Ruff as well.